src/test/regress/expected/multi_index_statements.out

```diff
@@ -280,78 +219,71 @@ DROP INDEX index_test_hash_index_a_b;
 DROP INDEX index_test_hash_index_a_b_partial;
 -- Verify that we can drop indexes concurrently
 DROP INDEX CONCURRENTLY lineitem_concurrently_index;
+-- Verify that all indexes got created on the coordinator node and on the workers
+-- by dropping the indexes. We do this because in different PG versions,
+-- the expression indexes are named differently
+-- and, being able to drop the index ensures that the index names are
+-- proper
+CREATE OR REPLACE FUNCTION drop_all_indexes(table_name regclass) RETURNS INTEGER AS $$
+DECLARE
+  i RECORD;
+BEGIN
+  FOR i IN
+    (SELECT indexrelid::regclass::text as relname FROM pg_index
+     WHERE indrelid = table_name and indexrelid::regclass::text not ilike '%pkey%')
+  LOOP
+    EXECUTE 'DROP INDEX ' || i.relname;
+  END LOOP;
+RETURN 1;
+END;
+$$ LANGUAGE plpgsql;
+SELECT drop_all_indexes('public.lineitem');
+ drop_all_indexes
+---------------------------------------------------------------------
+                1
+(1 row)
+
+SELECT drop_all_indexes('index_test_range');
+ drop_all_indexes
+---------------------------------------------------------------------
+                1
+(1 row)
+
+SELECT drop_all_indexes('index_test_hash');
```

Comment:
running this already fails on the master branch due to https://github.com/citusdata/citus/issues/5904